app/src/main/java/com/google/samples/apps/sunflower/PlantDetailFragment.kt

```diff
@@ -60,10 +62,16 @@ class PlantDetailFragment : Fragment() {
             setLifecycleOwner(this@PlantDetailFragment)
             fab.setOnClickListener { view ->
```

Comment:
`view` which is local will never be used, use  `_` instead.

Reply:
Got it.